src/main/java/seedu/address/model/VersionedAddressBook.java

```diff
@@ -19,7 +22,6 @@ public class VersionedAddressBook extends AddressBook {
      * Constructs a state list for the address book and saves the initial copy.
      */
     public VersionedAddressBook(ReadOnlyAddressBook addressBook) {
-        super(addressBook);
```

Comment:
I deleted the call to super class constructor as it is not needed.
It copies the content from the passed addressBook to this which is already done by line 26.